app/help/en/faq.md

```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+Over time, Steemit will allow advertisers and bloggers to promote content by buying and burning SBD/STEEM. Steemit can benefit from sales of Steem to advertisers in the cryptocurrency markets or offer advanced services to these advertisers and bloggers.
 
-Steemit, Steemit.com and Steemit, Inc. are all names for a privately-owned company and website offering people a secure way to interact with the community network. Steemit is simply an interface to view the blockchain content of Steem.
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="Can_I_invest_in_Steemit"></a>
+## Can I invest in Steemit?
 
-<a href="/faq.html#Table_of_Contents">Table of Contents</a>
-<a class="anchor" name="Can_I_earn_digital_points_on_Steemit_How"></a>
-## Can I earn digital points on Steemit? How?
+Steemit, Inc. is a privately held company and is not available for public investment.
 
-You can earn digital points on Steemit by:
+Though not considered an investment, you can speculate on the platform's success by purchasing STEEM tokens. You can power up these tokens into STEEM Power, which grants more influence in the Steem platform.
 
-**Blogging/Posting** - By sharing your original and unique posts, you can get upvotes by community members. Depending on the upvotes you receive, you will get a portion of that day’s total payout.
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="What_does_Steemit_s_development_roadmap_look_like"></a>
+## What does Steemit’s development roadmap look like?
 
+You can view the 2017 Roadmap here:
+https://steemit.com/steemit/@steemitblog/steemit-2017-roadmap
 
-**Curating/Voting** - If you discover a post and upvote it before it becomes popular, you will receive a curation reward. The reward amount will depend on the amount you have vested, called Steem Power.
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="Am_I_allowed_to_use_the_Steemit_logo"></a>
+## Am I allowed to use the Steemit logo?
 
-<a href="/faq.html#Table_of_Contents">Table of Contents</a>
-<a class="anchor" name="Is_Steemit_a_scam"></a>
-## Is Steemit a scam?
+Currently, the Steem and Steemit logos are the same and is free to use. In the future, Steemit, Inc. will have its own logo so that it can be distinguished from Steem. The Steemit logo will be proprietary while Steem and its three S-shaped squiggles will remain open for public use.
 
-Steemit is not a scam. A scam begins with you having to give money to someone else. Steemit is not a middleman to any of the digital points allocations users receive. Steemit doesn't ask you to invest anything.  The Steem blockchain and points system is entirely its own entity, similar to Bitcoin, and it should be treated with a high level of due diligence for anyone looking to leverage its digital points as cryptocurrency.
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
```

Comment:
Please add a note suggesting that people avoid use of the term "Steemit" in their own products and instead refer to the Steem Blockchain or Steem Platform.  Steemit means steemit.com.